### PR TITLE
add --show-output to document

### PR DESCRIPTION
**Pull request by cnmade**

### CODE REVIEW

**Comment by cnmade** [comment]
see document : https://doc.rust-lang.org/book/ch11-02-running-tests.html#showing-function-output

File: src/doc/src/commands/cargo-test.md [hunk 1]
```diff
@@ -310,6 +310,17 @@ specified with the <code>CARGO_TARGET_DIR</code> environment variable, or the
...
+ 
+If we want to see printed values for passing tests as well, we can tell Rust
+to also show the output of successful tests at the end with `--show-output`.
+
+```console
+$ cargo test -- --show-output
+```
+
+When we run the tests in Listing 11-10 again with the `--show-output` flag, we
+see the following output.
```
Comment (weihanglo) [inline src/doc/src/commands/cargo-test.md:323]: Here are things I am concern about:

- Under the section "Display Options", we have mentioned the `--nocapture` flag, which is somehow overlapped with `--show-output`. The change you made here might be better moving there and reword a bit, though personally, I am not sure where `--show-output` is that worthy to mention.
- Throughout the whole cargo-test documentation, the pronoun "you" is more common than "we". Could you change it to "you" for consistency? 
- cargo-bench gets [a similar paragraph](https://github.com/rust-lang/cargo/blob/master/src/doc/man/cargo-bench.md#display-options) about the test harness option `--nocapture`. You would like to update it as well.

**Review by weihanglo — COMMENTED** [review]
Thanks for the contribution. It's great to see people trying to improve on documentations!

The doc you modified here is a generated one. The original one you should change is located [here](https://github.com/rust-lang/cargo/blob/master/src/doc/man/cargo-test.md#display-options). The idea is that we reuse the man page for both CLI and HTML references. You can follow the guide about [how to work on Cargo's documentation](https://github.com/rust-lang/cargo/tree/master/src/doc).

**Comment by cnmade** [comment]
就这样吧。反正rust 文档稀烂，无力改进，爱谁谁吧。

**Pull request closed without merging**